docassemble/juvenilesealing/data/questions/juvenile-sealing-text-10.yml

```diff
@@ -90,58 +90,30 @@ content: |
   ${ docket_sheets_blurb } ${ read_more_docket_sheets }
 ---
 id: certified_copies_page_text_question
-field: about_docket_sheets
+field: request_docket_sheets
 # TODO: Are there courthouse instructions online for requesting
 #    docket sheets? A: No
-# [text](${ url_action('about_docket_sheets') })
+# [text](${ url_action('request_docket_sheets') })
 terms:
   a copy of your Massachusetts criminal records: ${ records_popover }
   your Massachusetts CORI report and on your juvenile records: ${ records_popover }
-question: About Docket Sheets
+question: How to Get Docket Sheets
 subquestion: |
-  ### What are docket sheets?
-
-  ${docket_sheets_blurb} Docket
-  sheets are papers from the courthouse that say
-  what happened in a case every time you went
-  to court for that case.
-  
-  ### What's different about a certified copy of the docket sheets?
-
-  A certified copy of the sheets
-  will have a special mark on it that looks like the
-  courthouse's seal. It'll either be a colored stamp or
-  an area where the paper is raised up in the shape of the
-  seal.
-
-  ### Why can sealing be a problem?
-
-  If you seal your records you won't be able to get
-  certified copies of your docket sheets until you
-  unseal them again. That can sometimes take a long time.
-  If you need to show your docket
-  sheets to a judge at an immigration hearing or if you
-  apply for some kinds of jobs, it might take too long
-  to unseal your records
-  and get those docket sheets.
-
   ## Getting certified copies
 
+  Having
+  {a copy of your Massachusetts criminal records}
+  can make it easier to make sure you get all your docket
+  sheets because they will have the docket numbers for
+  your cases. Every case has its own docket
+  sheets and those sheets are kept at the courthouse
+  that handled that case.
+
   Docket sheets for a case are kept at the courthouse
   that handled that case. To get docket sheets for a
   case, you might need to know the docket number of
-  that case. You can find those docket numbers on 
+  that case. You can find those docket numbers on
   {your Massachusetts CORI report and on your juvenile records}.
-  
-  ### Getting certified copies in person
-  
-  Getting copies in person is the easiest and fastest
-  way to do this. For each case, go to the clerk’s
-  office in the courthouse that handled the case and say,
-  "I'd like certified copies of my docket sheets
-  for all of my cases." You have to do it
-  at each courthouse where you had a case. You might need
-  to know the docket numbers of your cases.
```

Comment:
Ok, this one I think I'm correct about (though that's how I felt about all of them), and can also be addressed in a future PR - what's the reasoning behind removing this text? I can't find other places that contain this information.

Reply:
Really good catch, thank you. I'll fix. I think I erased this accidentally.